etc/releasing.md

```diff
@@ -331,6 +331,16 @@ Commit the updates to `CHANGELOG.md`.
 git commit -m 'Update CHANGELOG for X.Y.Z'
 ```
 
+### Update GitHub Webhook
```

Comment:
Recommend moving this script down immediately before the "Update releases/stable" step to avoid updating the webhook before additional changes may be committed prior/during the release.

Reply:
Poke, not yet resolved.

Reply:
Oops. Good catch. I remember relocating this section, but do not know how I lost the changes. Updated.